pre-build/action.yml

```diff
@@ -80,6 +80,7 @@ runs:
         BOTTLES_DIR: ${{ inputs.bottles-directory }}
         MACOS_VERSION: ${{ steps.setup-cache.outputs.macos_version }}
         ARCH: ${{ steps.setup-cache.outputs.arch }}
+        RUNNER_OS: ${{ runner.os }}
```

Comment:
This is predefined by default: https://docs.github.com/en/actions/reference/workflows-and-actions/variables#default-environment-variables

```suggestion
```

Reply:
Yeah, it's funny that GitHub's own AI suggestion here doesn't know which variables are built-in...

(Technically, adding the env here is subtly wrong: it's predefined *and* the predefined version always takes precedence, so setting it is misleading about where the value gets sourced from.)